### PR TITLE
Adapt SNMP support

### PR DESCRIPTION
**Pull request by phox142**
##### SUMMARY
- add support for trap-interfaces in API paths ``snmp``
- add support for API paths ``snmp community`` 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- snmp
- snmp community

### CODE REVIEW

File: changelogs/fragments/159-snmp_community.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - api_modify - add support for trap-interfaces in API paths ``snmp`` (https://github.com/ansible-collections/community.routeros/pull/159).
+  - api_modify - add support for API paths ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/159).
```
Comment (felixfontein) [inline changelogs/fragments/159-snmp_community.yml:3]:
```suggestion
  - api_modify - add support for API path ``snmp community`` (https://github.com/ansible-collections/community.routeros/pull/159).
```

File: changelogs/fragments/159-snmp_community.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - api_modify - add support for trap-interfaces in API paths ``snmp`` (https://github.com/ansible-collections/community.routeros/pull/159).
```
Comment (felixfontein) [inline changelogs/fragments/159-snmp_community.yml:2]:
```suggestion
  - api_modify - add support for ``trap-interfaces`` in API path ``snmp`` (https://github.com/ansible-collections/community.routeros/pull/159).
```

**Review by felixfontein — COMMENTED** [review]
Thanks also for this contribution!

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@phox142 thanks again :)

**Comment by phox142** [comment]
@felixfontein I'm trying to help the one whose development I'm using (time) ;)

**Pull request merged**